CMakeLists.txt

```diff
@@ -617,7 +617,7 @@ if (LIBRAW_INSTALL)
     if(NOT BUILD_SHARED_LIBS AND "${CMAKE_CXX_SIMULATE_ID}" STREQUAL "MSVC")
         message("ClangCl does not support pdb generation with static libraries")
     elseif(MSVC)
-        install(FILES ${PROJECT_BINARY_DIR}/raw.pdb ${PROJECT_BINARY_DIR}/raw_r.pdb
+        install(FILES ${PROJECT_BINARY_DIR}/raw${CMAKE_DEBUG_POSTFIX}.pdb ${PROJECT_BINARY_DIR}/raw_r${CMAKE_DEBUG_POSTFIX}.pdb
```

Comment:
The CMake docs https://cmake.org/cmake/help/latest/variable/CMAKE_CONFIG_POSTFIX.html don't suggest that this is only set for matching build types. Which makes sense to me as having this information available in general is useful for multi-config generators.
@hartcw Any advice since you added pdb installation in https://github.com/LibRaw/LibRaw-cmake/commit/9e4991fdbdb554c9b8f80574705a795198e132de?